roles/testnode/tasks/setup-ubuntu.yml

```diff
@@ -12,6 +12,6 @@
   lineinfile:
     dest: /etc/lvm/lvm.conf
     regexp: "# types ="
-    line: 'types = [ "rbd", 1024 ]'
+    line: 'types = [ "rbd", 16 ]'
```

Comment:
Originally set here for posterity: https://github.com/ceph/ceph-qa-chef/pull/12/files